docs/product-editor-development/product-editor-extensibility-guidelines.md

```diff
@@ -0,0 +1,97 @@
+# Product Editor Extensibility Guidelines
+
+## Form Extensibility
+
+Our research shows that merchants think in terms of tasks and goals while creating new products in Woo. For example, adding or editing a product price is a separate task from uploading images or managing inventory.
+
+For this reason, the new product form features tabs that correspond with these tasks. Simply speaking, they are separate views where different form features live. As a developer, you can add your extension to any of these views and offer merchants the information and tools they need to create better, more successful products.
+
+There are several ways to extend the new product form: from a single field to a whole section containing multiple fields and tables. These extension points are tightly linked to the form structure, giving you plenty of freedom to create a great user experience.
+
+Like everything in the new product form, each extension point is a separate block. However, each existing block can be extended the same as the Block Editor, so for the sake of a consistent and smooth user experience, try to integrate your extension into an existing block before creating a new one.
+
+## Product form tabs
```

Comment:
A picture is worth a thousand words... it would be nice to have a screenshot of the form with the tabs.

Reply:
This image exists [here](https://github.com/woocommerce/woocommerce/tree/trunk/docs/product-editor-development#product-editor-development-handbook) already. Should we add the same image, perhaps? Or something different?

Reply:
I'd probably just have a screenshot of the form, not annotated. Maybe an animated GIF that cycles between the tabs>